README.md

```diff
@@ -1,70 +1,77 @@
-# Getting Started with Create React App
+# react-capstone
+A Metrics-webapp is a mobile web application to check a list of corona virus statistics based on country (numeric values). It's built with React and Redux.
```

Comment:
#### SIDE NOTE:
In your future projects, I'd recommend being consistent and sticking to the **Commit messages** and **branch name** conventions:
- As highlighted by Microverse in this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), commit messages should be **capitalized and imperative**.
- Your branch and Pull Request names (**feature** and **Feature**) are also too generic. They are not descriptive of the feature you just built